backend/app/services/adapters/task_kinds/converters.py

```
@@ -75,6 +75,34 @@ def get_task_execution_workspace_path(task_crd: Task) -> str | None:
+) -> Kind | None:
+    team_ref = task_crd.spec.teamRef
+    team_ref_user_id = getattr(team_ref, "user_id", None)
+    if team_ref_user_id is not None:
+        team = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == team_ref_user_id,
+                Kind.kind == KindType.TEAM.value,
+                Kind.namespace == team_ref.namespace,
+                Kind.name == team_ref.name,
+                Kind.is_active == True,
```

Ruff (0.15.20)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)